kanopi/kanopi-pack
# Development Server CSS Placement

## Description
> As a developer, I need to offer a way to place CSS delivered through JS, in Dev mode, in a consistent location when needed by a project.

## Related Tickets

#39 

## Steps to Validate
1. Install the PR branch version in your project
2. Specify a valid CSS selector (i.e. `#some-id` ) in the KP configuration at `styles.devHeadSelectorInsertBefore`
3. Restart the Dev Server
4. Visit a page which uses the Dev server, verify the styles are in the header (only in the Inspector, not page source)
5. Make a change and verify the new styles are loaded, and injected before the specified element (again in Inspector)
